HealthcareSystem/PatientWebApp/DTOs/ActionBenefitDTO.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace PatientWebApp.DTOs
+{
+    public class ActionBenefitDTO
+    {
+        public int Id { get; set; }
+        public int PharmacyId { get; set; }
+        public string Subject { get; set; }
+        public string Message { get; set; }
+        public bool IsPublic { get; set; }
```

Comment:
Why would you need this in the DTO? It's not like the patient should be able to see any that aren't public.

Reply:
Fixed in a5ad64709381a12cb71e9eaa5b25cbba14c76a01.